### PR TITLE
[TwigBridge] add missing unit tests (AppVariable)

### PR DESCRIPTION
**Pull request by FlorianLB**
Just add few missing unit tests.

---

| Q | A |
| --- | --- |
| Bug fix? | [no] |
| New feature? | [no] |
| BC breaks? | [no] |
| Deprecations? | [no] |
| Tests pass? | [yes] |
| Fixed tickets | [n/a] |
| License | MIT |
| Doc PR | [n/a] |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Bridge\Twig\Tests\DataCollector;
+
+use Symfony\Bridge\Twig\DataCollector\TwigDataCollector;
+use Twig_Profiler_Profile;
+
+class TwigDataCollectorTest extends \PHPUnit_Framework_TestCase
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php]: this should not be in this PR. It is unrelated (and incomplete)

Comment (FlorianLB) [inline src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php]: Incomplete but at least we test instantiation. I can remove it if you want.

Comment (WouterJ) [inline src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php]: it does not test anything. Such things should be checked using something like PHP codesniffer, PHP CS fixer or other linters, tests are testing behaviour/public API.

Comment (FlorianLB) [inline src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php]: I don't completely agree but no worries, I removed it (and fix a CS typo).

File: src/Symfony/Bridge/Twig/Tests/AppVariableTest.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+use Symfony\Component\HttpFoundation\Session\Session;
+
+class AppVariableTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var AppVariable
+     */
+    private $appVariable;
+
+    public function setUp()
```
Comment (jakzal) [inline src/Symfony/Bridge/Twig/Tests/AppVariableTest.php]: Should be `protected`.

Comment (FlorianLB) [inline src/Symfony/Bridge/Twig/Tests/AppVariableTest.php]: Fixed.

**Pull request closed without merging**